counter.py

```diff
@@ -3,38 +3,23 @@
 from os.path import exists
 import sys
 from pickle import dump, load
+import codecs
 
 
 def update_counter(file_name, reset=False):
-    """ Updates a counter stored in the file 'file_name'
-
-    A new counter will be created and initialized to 1 if none exists or if
-    the reset flag is True.
-
-    If the counter already exists and reset is False, the counter's value will
-    be incremented.
-
-    file_name: the file that stores the counter to be incremented.  If the file
-    doesn't exist, a counter is created and initialized to 1.
-    reset: True if the counter in the file should be rest.
-    returns: the new counter value
-
-    >>> update_counter('blah.txt',True)
```

Comment:
Try to leave in the test cases. If your code passes those, we know things are going as expected.